Src/java/README.md

```diff
@@ -1,12 +1,17 @@
 # Overview
 
-This is a multi-module project for CQL java libraries and proof-of-concept applications.
+This is a multi-module project for CQL java libraries and tooling applications.
 It contains the following sub-projects:
 
 * **cql:** generates and builds Java lexers, parsers, listeners, and visitors using the CQL ANTLR4 grammar
-* **quick:** generates and builds Java classes based on the QUICK XML schema
+* **model:** generates and builds Java classes based on the ELM Model Info schema and CQL base type system
 * **elm:** generates and builds Java classes based on the ELM XML schema
-* **cql-to-elm:** generates Expression Logical Model (ELM) XML from CQL source
+* **qdm:** contains schema and model info resources for QDM (4.2, 5.0, 5.0.1, 5.0.2, 5.3)
+* **quick:** contains schema and model info resources for QUICK and FHIR, DSTU2 (1.0.2), and STU3 (1.4, 1.6, 1.8, and 3.0.1)
```

Comment:
I think we eventually need to give fhir its own space.  (Just a comment; nothing to be acted on right now).